docs/platforms/javascript/common/performance/instrumentation/custom-instrumentation.mdx

```diff
@@ -16,21 +16,153 @@ To get started, import the SDK.
+
+This is a tradeoff that we have made to ensure that the data that is captured is accurate and reliable. If you need to capture a more complex hierarchy of spans, you can opt-out of this behavior by setting `parentSpanIsAlwaysRootSpan: false`:
+
+```javascript
+Sentry.init({
+  parentSpanIsAlwaysRootSpan: false,
+});
+```
+
+This will revert to use the full hierarchy behavior, where spans are children of the currently active span. However, this may lead to incorrect data in the case of multiple parallel asynchronous operations - it is up to you to ensure there are not multiple parallel asynchronous operations that start spans in this case.
+
+</PlatformCategorySection>
+
+## Span Starting Options
+
+The following options can be used for all span starting functions:
+
+| Option             | Type                        | Description                                                         |
+| ------------------ | --------------------------- | ------------------------------------------------------------------- |
+| `name`             | `string`                    | The name of the span.                                               |
+| `op`               | `string`                    | The operation of the span.                                          |
+| `startTime`        | `number`                    | The start time of the span.                                         |
+| `attributes`       | `Record<string, Primitive>` | Attributes to attach to the span.                                   |
+| `onlyIfParent`     | `boolean`                   | If true, ignore the span if there is no active parent span.         |
+| `forceTransaction` | `boolean`                   | If true, ensure this span shows up as transaction in the Sentry UI. |
+
+Only `name` is required, all other options are optional.
+
+## Starting an Active Span (`startSpan`)
+
+For most scenarios, we recommend to start active spans with `Sentry.startSpan()`. This will start a new span that is active in the provided callback, and will automatically end the span when the callback is done. The callback can be synchronuous, or asynchronous (a promise). In the case of an asynchronous callback, the span will be ended when the promise is resolved or rejected. If the provided callback errors or rejects, the span will be marked as failed.
 
 <PlatformContent includePath="performance/start-span" />
 
-## Start Inactive Spans
+## Starting an Active Span with Manual End (`startSpanManual`)
+
+Sometimes, you do not want the span to be ended automatically when the callback is done. In this case, you can use `Sentry.startSpanManual()`. This will start a new span that is active in the provided callback, but will not automatically end the span when the callback is done. You have to manually end the span by calling `span.end()`.
```

Comment:
```suggestion
Sometimes, you do not want the span to be ended automatically when the callback is done. In this case, you can use `Sentry.startSpanManual()`. This will start a new span that is active in the provided callback, but will not be automatically ended when the callback is done. You have to manually end the span by calling `span.end()`.
```